### PR TITLE
Configure the release-next branch to run with OCP 4.1 and 4.2

### PR DESCRIPTION
**Pull request by alanfx**

### CODE REVIEW

**Comment by alanfx** [comment]
@stevekuznetsov created this PR based on our Slack conversation. I didn't see anything in the postsubmits that was needed to be changed, but let me know if you think I am wrong. If you think this changes are OK, I'll add the changes for the other Knative repos as well. Thanks!

File: ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-next__variant.yaml [hunk 1]
```diff
@@ -0,0 +1,223 @@
+tag_specification:
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-next__variant.yaml]: @smarterclayton did we remove support for `__variant`?

**Comment by stevekuznetsov** [comment]
Actually i lied -- looks like @alanfx you didn't run the generator :)

**Comment by alanfx** [comment]
@stevekuznetsov I didn't! Can you be more specific about what you think I should do?

**Comment by alanfx** [comment]
Sorry @stevekuznetsov, I just can't get this to work for me whether I use the generators or manually edit the files. You have any other suggestions?

**Comment by alanfx** [comment]
Sorry @stevekuznetsov, I just can't get this to work for me even after the fix. You have any other suggestions?

**Comment by alanfx** [comment]
@stevekuznetsov it looks like the job configs are only generated for the `__variant`, but the original ones are missing now. I basically copied the existing config file, changed the names and OCP version, and then ran the `docker run -it -v "$(pwd)/ci-operator:/ci-operator:z" registry.svc.ci.openshift.org/ci/ci-operator-prowgen:latest --from-dir /ci-operator/config --to-dir /ci-operator/jobs` command. If there is something else I should do, please let me know!

**Pull request merged**